README.md

```diff
@@ -1,10 +1,60 @@
-# Website
+# Project Layout
+
+### The project is broken down into 4 sections
+- Concepts - General Uniswap information or concepts useful for using Uniswap products, such as *Liquidity* and *Fees*
+- Contracts - The Uniswap smart contracts such as *Permit2*, or groupings such as the V3 Contracts
+- SDKs - The Uniswap SDKs such the *v3-sdk*
```

Comment:
maybe good to mention the widget here to show this goes beyond the literal low-level SDK's?

```suggestion
- SDKs - Uniswap integrations such as the *v3-sdk* and the *Swap Widget*
```